src/www/services_ntpd.php

```diff
@@ -99,7 +99,7 @@
     }
 
     // swap fields, really stupid field usage which we are not going to change now....
-    foreach (array('kod', 'limited', 'nomodify', 'nopeer', 'notrap') as $fieldname) {
+    foreach (array('kod', 'limited', 'nomodify', 'nopeer', 'notrap', 'noquery') as $fieldname) {
```

Comment:
or am I missing this one? I mean I agree straightening out the GUI but we cannot change the config render for existing entries in user configs as that would flip their default (unless that is the intention here).

Reply:
Note the missing migration note. If users have `noquery` currently set in config.xml, it should be unset via migration. Then these fields appear ticked and "enabled by disabling" 

That is exactly the braindead way all those other tripple-inverted enabled by disabling fields are (that `/*This one works backwards */` code in `ntpd.inc` is - and I did not want to make any special case for this one. Would confuse matters even further.

Reply:
since we don't have a migration in place here can we do the unfathomable and call this field 'noquery2'?

Reply:
And keep the rest of the _really stupid_, _backwards_ etc. as is in this PR? Tempting indeed. Just need to check whether there are not other references to the field elsewhere.

Reply:
Well, kill the old `noquery` with fire. We might get away with this during a minor change due to security concerns.